packages/sdk/package.json

```diff
@@ -35,7 +35,10 @@
     "@nomad-xyz/contracts-core": "workspace:^",
     "@nomad-xyz/multi-provider": "workspace:^",
     "axios": "^0.27.2",
```

Comment:
I'd prefer just keeping cross-fetch over axios if that one works better for us. Not a big deal for now tho. Noticed we only use axios in `getProof()`

Reply:
I just decided to include those changes in this PR, so I'm not mixing code with a release PR